GPy/kern/src/multioutput_derivative_kern.py

```diff
@@ -23,7 +23,7 @@ def gradients_X(self,dL_dK, X, X2=None):
         return self.fg(dL_dK, X, X2=X2)
 
     def get_gradient(self):
-        return self.base_kern.gradient.copy()
```

Comment:
would it be possible to just make `gradient` point to the `base_kern.gradient` in where necessary?

e.g.:

```python
@property
def gradient(self):
    return self.base_kern.gradient
```

@gradient.setter
def gradient(self, gradient):
   self.base_kern.gradient = gradient
```

this would mean, there are no gradients on the multioutput_derivative_kern itself, though, so maybe this needs further discussion?

Reply:
Actually this is what you tried to do in 
https://github.com/SheffieldML/GPy/pull/678/files/ee7f23869bf336d547616a2faf6d5b7030d24835#diff-59ae49a0a728fd82c52c5f8793797e72R21
anyway right?

Reply:
Yes, I tried. I actually just forgot to remove those lines after my unsuccessful test on if this was possible. I'll remove this line! Thank you for noticing this!